Automattic/sensei
# Suppress theme without comments.php deprecation message

Fixes #6444.

### Changes proposed in this Pull Request
Block themes don't have a `comments.php` file. Some might have a `/patterns/comments.php` file, but not all. So, this PR just suppresses the message by using the solution in https://github.com/WordPress/gutenberg/pull/28128.

### Testing instructions
1. Activate a block theme (e.g. Course or Twenty Twenty Three).
2. In _Sensei LMS_ > _Settings_ > _Lessons_, enable _Allow Comments for Lessons_.
3. View a lesson as a student.
4. Ensure no theme deprecation notice is displayed / logged.
